jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/FifoPanel.kt

```diff
@@ -41,21 +41,28 @@ class FifoPanel {
         groupId.emptyText.text = message("sqs.required.empty.text")
     }
 
-    fun validateFields(): ValidationInfo? = when {
-        deduplicationId.text.length > MAX_LENGTH_OF_FIFO_ID -> {
-            ValidationInfo(message("sqs.message.validation.long.id"), deduplicationId)
+    fun validateFields(): List<ValidationInfo> = listOfNotNull(
+        when {
```

Comment:
the list of whens seems rather gross, can you just extract each when to its own method

Reply:
yep, changed